cinema/serializers.py

```diff
@@ -68,13 +78,77 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available"
```

Comment:
The `MovieSessionListSerializer` includes the `tickets_available` field. Ensure that this field is correctly implemented in the `MovieSession` model to accurately reflect the number of available tickets.